stable/prometheus-mongodb-exporter/README.md

```diff
@@ -62,8 +64,3 @@ podAnnotations:
 | `serviceMonitor.namespace` | The namespace where the Prometheus Operator is deployed | `` |
 | `serviceMonitor.additionalLabels` | Additional labels to add to the ServiceMonitor | `{}` |
 | `tolerations` | List of node taints to tolerate  | `[]` |
-
-## Limitations
```

Comment:
Removed the limitations statement since I'm using the mongodb_exporter of this chart to connect to a TLS 1.2 only enabled MongoDB cluster.

mongodb_exporters [documentation](https://github.com/percona/mongodb_exporter#flags) refers also to the official [connection options](https://docs.mongodb.com/manual/reference/connection-string/#connections-connection-options) where TLS is possible and supported by the drivers.